### ISSUE TITLE
Different between instainsane and instashell

### ISSUE THREAD

**Issue by Franchium** [thelinuxchoice/instainsane#19]
what is the different between instainsane and instashell ? both of it is currently still develop isnt ?

**Comment by greyhat00sudo** [comment]
Yup instainsane requires root and instashell doesn't and instainsane is better according to me
